lib/jna/clink_jna.cc

```diff
@@ -26,20 +27,30 @@
 extern "C" {
 #endif
 
+tfrt::HostContext *jna_host_context =
+    clink::CreateHostContext("mstd", tfrt::HostAllocatorType::kLeakCheckMalloc)
```

Comment:
Should we use `kMalloc` as the default allocator in the production code for best possible performance?

`kLeakCheckMalloc` can be used as the default allocator in test code. And it can be used as optional allocator in the production code when users explicitly enables it to debug e.g. memory leak.

Reply:
Got it. Maybe we can change it to as below later when we work on the PR for automatic memory leak check.
```suggestion
#ifndef NDEBUG
    clink::CreateHostContext("mstd", tfrt::HostAllocatorType::kMalloc)
#else
    clink::CreateHostContext("mstd", tfrt::HostAllocatorType::kLeakCheckMalloc)
#endif
```